sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -308,6 +332,16 @@ public static void notifyUserConfirmation(@UserConfirmationDef int userConfirmat
         return getInstance().getInstanceLastSessionCrashReport();
     }
 
+    /**
+     * Check whether in the last session the memory warning.
+     *
+     * @return future with result being <code>true</code> if memory running was critical, <code>false</code> otherwise.
+     * @see AppCenterFuture
+     */
+    public static AppCenterFuture<Boolean> hadMemoryWarningInLastSession() {
```

Comment:
Did PM sign off on this name? We should use the same name as iOS (which is currently this but may change). I propose `receivedMemoryWarningInLastSession` for both iOS and Android.